### PR TITLE
Updating Android FAB

### PR DESCRIPTION
**Pull request by leticiarossi**

### CODE REVIEW

**Review by mingjane-work — APPROVED** [review]
LGTM.

As a reminder, you don't need to pull into my repo for reviews: add me when you update the Android repo.

File: components/FAB/Android/Android-FAB.md [hunk 1]
```diff
@@ -1,159 +1,274 @@
 <!--docs:
-title: "Floating action button"
+title: "Floating action buttons"
 layout: detail
 section: components
 excerpt: "A floating action button (FAB) represents the primary action of a screen."
-iconId: 
+iconId: button
```
[the remark this answers is not in the document]
Comment (leticiarossi) [reply components/FAB/Android/Android-FAB.md:6]: this is unchanged from the existing doc, don't want to change the existing metadata so I think it's fine to leave as is.

**Comment by leticiarossi** [comment]
> LGTM.
> 
> As a reminder, you don't need to pull into my repo for reviews: add me when you update the Android repo.

sorry I'm confused, should we not be submitting PRs anymore?

**Comment by mingjane-work** [comment]
> > LGTM.
> > As a reminder, you don't need to pull into my repo for reviews: add me when you update the Android repo.
> 
> sorry I'm confused, should we not be submitting PRs anymore?

Not into my repo.

**Review by akkapon75 — APPROVED** [review]

**Pull request closed without merging**